For some reasons, cmake find_file() fails on cross-build (only).
Let's bypass cmake path computation.

--- src/runtime/kwalletd/CMakeLists.txt.ORIG
+++ src/runtime/kwalletd/CMakeLists.txt
@@ -78,7 +78,7 @@
     )
 endif(Gpgmepp_FOUND)
 
-find_file(kwallet_xml org.kde.KWallet.xml HINTS ${CMAKE_CURRENT_SOURCE_DIR}/../../api/KWallet)
+set(kwallet_xml ${CMAKE_CURRENT_SOURCE_DIR}/../../api/KWallet/org.kde.KWallet.xml)
 
 qt5_add_dbus_adaptor( kwalletd_SRCS ${kwallet_xml} kwalletd.h KWalletD )
 
